src/hotspot/share/opto/superword.cpp

```diff
@@ -2042,18 +2084,21 @@ class PacksetGraph {
   }
 };
 
-// The C2 graph (specifically the memory graph), needs to be re-ordered.
+// We want to replace the packed scalars from the PackSet and replace them
+// with vector operations. This requires scheduling and re-ordering the memory
+// graph. We take these steps:
 // (1) Build the PacksetGraph. It combines the dependency graph with the
 //     packset. The PacksetGraph gives us the dependencies that must be
 //     respected after scheduling.
 // (2) Schedule the PacksetGraph to the memops_schedule, which represents
 //     a linear order of all memops in the body. The order respects the
 //     dependencies of the PacksetGraph.
 // (3) If the PacksetGraph has cycles, we cannot schedule. Abort.
-// (4) Use the memops_schedule to re-order the memops in all slices.
-void SuperWord::schedule() {
+// (4) Apply the vectorization, including re-ordering the memops and replacing
+//     packed scalars with vector operations.
+bool SuperWord::schedule_and_apply() {
   if (_packset.length() == 0) {
-    return; // empty packset
+    return false; // empty packset
```

Comment:
With the change above, the comment is no longer necessary (somehow I cannot make a suggestion here to directly apply).